src/frontend/containers/withFirefighters/index.js

```diff
@@ -49,13 +50,20 @@ export default ChildComponent =>
       this.loadResponseIntoState(response);
     };
 
+    updateFirefighterDuty = async (id, isOnDuty) => {
+      const response = await updateFirefighterDuty(id, isOnDuty);
```

Comment:
Reminder for the future: never ever use `async/await` generators without `try/catch` (I also didn't use that but hey, we should)

Reply:
Next browser updates will explode everything if a uncaught promise error happens